components/HomeHero.tsx

```diff
@@ -3,13 +3,57 @@ import { EXCHANGES, UI } from '@/constants';
 import classNames from 'classnames';
 import { Button } from './Button';
 
+type HeroButtonProps = {
+  href: string,
+  ariaLabel: string,
+  label: string,
+}
+
+function HeroButton({
+  href,
+  ariaLabel,
+  label,
+}:HeroButtonProps) {
+  return (
+    <a
+      href={href}
+      target="_blank"
+      rel="noreferrer"
+      aria-label={ariaLabel}
+      className={classNames('mt-4', 'tablet:mt-12', 'desktop:mt-8')}
+    >
+      <Button size="large" className="font-sans">
+        {label}
+      </Button>
+    </a>
+  );
+}
+
+const heroButtons: Array<HeroButtonProps> = [
+  {
+    href: 'https://oxen.io/blog/development-is-transitioning-to-session-token/',
+    ariaLabel: 'Link to Oxen blog post about the migration',
+    label: 'Learn more about the migration',
+  },
+  {
+    href: 'https://swap.oxen.io/',
+    ariaLabel: 'Link to Session Token Swap Program',
+    label: 'Participate in the Session Token Swap Program',
+  },
+  {
+    href: 'https://token.getsession.org/oxen-coin-claims',
+    label: 'Learn more about the Oxen Coin Claims Program',
+    ariaLabel: 'Link to Oxen Coin Claims Program',
+  },
+] as const;
+
 export function HomeHero() {
   return (
     <div
       className={classNames(
-        'relative w-full flex justify-center items-center overflow-x-hidden',
+        'relative w-full flex justify-center items-center overflow-x-hidden pt-96',
```

Comment:
Slightly overkill but you can't see the heading properly at 320px. Don't worry about the button overflow but at least the content should be readable 🙏

<img width="372" alt="Screenshot 2024-06-04 at 09 59 39" src="https://github.com/oxen-io/oxen-website/assets/14887287/7ee0e474-20dd-496f-a013-49e3004717ba">

Reply:
Redone styles